data/sources/00_areatree

```diff
@@ -239,7 +239,7 @@
     3901:Kiosk:
     3902:Seglerheim:
     3904:Bootshaus:
-  4:Wissenschaftszentrum Weihenstephan (WZW Freising)|WZW Freising:wzw[campus]
+  4:Campus Weihenstephan:wzw[campus]
```

Comment:
```suggestion
  4:Campus Weihenstephan (Freising)|Weihenstephan:wzw[campus]
```
I think we should still include "Freising", because afaik it is commonly used